tests/framework/utils.py

```diff
@@ -510,6 +510,33 @@ def get_cpu_percent(pid: int, iterations: int, omit: int) -> dict:
     return cpu_percentages
 
 
+def summarize_cpu_percent(cpu_percentages: dict):
```

Comment:
Please let me ask if it's fine to drop DEBUG feature (storing all data), just in case.

Reply:
> We have plans to properly implement this in follow ups (@pb8o already did it for the network latency test).

Sounds great. Thanks.